libs/embed/src/embed.ts

```diff
@@ -17,6 +17,8 @@ class Novu {
 
   private socketUrl?: string = '';
 
+  private theme?: Record<string, unknown>;
```

Comment:
It would be great if we could create an interface or type for the theme 😄

Reply:
Sure! I thought of the same but I personally saw no advantage of doing so as this only bloats the devDependencies of the embed lib and in the end the user only receives a JS file with no types anyway 😅